site/operational-guide/interactive.md

```diff
@@ -2,55 +2,56 @@
 title: Interactive Deployment
 menu_order: 20
 ---
+Since launching Weave commands in interactive mode is not amenable to automation and
+configuration management, it is recommended that launching Weave in interactive 
+mode be used for exploration and evaluation only. However, if Weave is launched in 
+this mode, and as long as Docker is configured to start on boot,the resulting 
+network will survive host reboots without the use of a systemd.
 
-This pattern is recommended for exploration and evaluation only, as
-the commands described herein are interactive and not readily amenable
-to automation and configuration management. Nevertheless, the
-resulting weave network will survive host reboots without the use of a
-systemd unit as long as Docker is configured to start on boot.
+### Bootstrapping
 
-### Bootstrap
-
-On initial peer:
+On the initial peer:
 
     weave launch
 
-### Add a Peer
+### Adding a Peer
 
-On new peer:
+On a new peer:
 
     weave launch <extant peers>
 
-Where `<extant peers>` means all peers in the network, initial and
+Where, 
+
+* `<extant peers>` indicates all peers in the network, initial and
 subsequently added, which have not been explicitly removed. It should
-include peers which are temporarily offline or stopped.
+include peers that are temporarily offline or stopped.
 
-You must then execute:
+To ensure that the new peer has joined the existing network, 
+execute the following:
 
     weave prime
 
-to ensure that the new peer has joined to the existing network; you
-_must_ wait for this to complete successfully before moving on to add
-further new peers. If this command blocks it means that there is some
+Before adding any new peers, you _must_ wait for this to complete. 
+If this command waits and does not exit, it means that there is some
 issue (such as a network partition or failed peers) that is preventing
 a quorum from being reached - you will need to [address
 that](/site/troubleshooting.md) before moving on.
 
-### Stop a Peer
+### Stopping a Peer
 
 A peer can be stopped temporarily with the following command:
 
     weave stop
 
-Such a peer will remember IP address allocation information on the
+A temporarily stopped peer will remember IP address allocation information on the
 next `weave launch` but will forget any discovered peers or
 modifications to the initial peer list that were made with `weave
 connect` or `weave forget`. Note that if the host reboots, Docker
-will restart the peer automatically.
+automatically restarts the peer.
 
-### Remove a Peer
+### Removing a Peer
 
-On peer to be removed:
+On the peer to be removed:
```

Comment:
inconsistency here: with the 'Adding' case which says "*a* peer". (FWIW, I prefer 'the').